src/main/java/validation/EmptyLineValidity.java

```diff
@@ -0,0 +1,60 @@
+package validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Check if the Line is valid or not.
```

Comment:
It checks the empty lines of code.
The code that contains consecutive empty lines would be invalid.